test/remote/gateways/remote_blue_snap_test.rb

```diff
@@ -5,11 +5,13 @@ def setup
     @gateway = BlueSnapGateway.new(fixtures(:blue_snap))
 
     @amount = 100
+    @fraudulent_amount = 3729
```

Comment:
I am of many conflicting opinions on this. One has to change specific settings in one's dashboard to trigger any fraud rules at all (as well as having fraud checking turned on manually). Those settings are reflected in the @fraudulent_card and the @fraudulent_amount, and in the recorded responses in the unit test (which, since the responses are recorded, doesn't care how much you actually send or what card you use). But I didn't want to overspecify the remote test, since there are other options for fraud generation and they aren't universalizeable. 

I guess I'd say I went with a more descriptive approach overall?

Reply:
Very interesting. I had no idea that so many steps were needed to trigger a fraud response. I had just assumed that the fraudulent card number was doing all of the work here. Your explanation makes sense, though, so thanks for sharing!

Reply:
This is probably a good case for including a little more inline documentation; I'll add some.